ckeditor.php

```diff
@@ -68,7 +68,7 @@ function __construct(){
 	public function registerPaths(){
 	    if ( !$this->pathsAdded ){
                 Dataface_Application::getInstance()->addHeadContent(
-                        '<script>window.CKEDITOR_BASEPATH='.json_encode($this->getBaseURL().'/lib/ckeditor/').';</script>'
+                        '<script src="'.($this->getBaseURL().'/lib/ckeditor/ckeditor.js').'"></script>'
```

Comment:
What happens if you don't make this change? The ckeditor.js should be loaded by the Dataface_JavascriptTool automatically in the ckeditor-widget.js file.  Adding it directly here has some down-sides.

Reply:
Without this change, the editor doesn't appear...

Reply:
Is there an error in the javascript console?

Reply:
no one

Reply:
but the scayt module autoload config doesn't work..

Reply:
Yes, I don't understand why. I'll take a look too.. Sorry !